docs/sdk/api/syncActions.md

```diff
@@ -432,3 +435,50 @@ client.execute(zonesRequests)
 .then(result => ...)
 .catch(error => ...)
 ```
+
+## `createSyncProductTypes(actionGroups)`
+
+> From package [@commercetools/sync-actions](/sdk/api/README.md#sync-actions).
+
+Creates a [sync action](/sdk/Glossary.md#sync-action) that allows to build API update actions for _zones_.
+
+#### Arguments
+
+1. `actionGroups` _(Array)_: A list of [action group](/sdk/Glossary.md#sync-action) in case some actions need to be _blacklisted_ or _whitelisted_.
+
+#### Usage example
+
+```js
+import { createSyncProductTypes } from '@commercetools/sync-actions'
+import { createClient } from '@commercetools/sdk-client'
+
+const syncZones = createSyncProductTypes()
```

Comment:
Zones again? Hmm maybe I am unaware of something. What are zones?

Reply:
same as above :P